website/source/docs/virtualbox/networking.html.md

```diff
@@ -10,6 +10,22 @@ description: |-
 
 # Networking
 
+## VirtualBox Host-Only Networks
+
+By default, private networks are [host-only networks](https://www.virtualbox.org/manual/ch06.html#network_hostonly),
+because those are the easiest to work with. In VirtualBox, since you can
+create multiple host-only networks, it is also possible to specify which
+host-only network you want to the Vagrant Virtualbox provider to use for
```

Comment:
Small typo fix.... remove `to`, and update/capitalize `VirtualBox`:

> host-only network you want ~to~ the Vagrant VirtualBox provider to use for

Reply:
Done in commits/9951656cb65d67a84c62ae86ae510ce607eee7f8.